backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/SubscriptionsController.java

```diff
@@ -130,33 +137,31 @@ public SubscriptionModel getSubscriptionById(
 		return subscriptionModelMapper.toModel(userId, subscription);
 	}
 
-	//TODO: will use JSON PATCH in later PR
 	@PatchMapping({ "/{subscriptionId}" })
 	@Operation(summary = "Update a subscription by ID")
 	@ApiResponse(responseCode = "204", description = "The request has been successfully processed.")
 	public void updateSubscriptionById(
-			@NotBlank(message = "userId must not be null or blank")
+			@NotBlank(message = "id must not be null or blank")
```

Comment:
```suggestion
			@NotBlank(message = "userId must not be null or blank")
```

Reason: the parameter name is `userId`. If you want to keep this line as `id`, then you should change the parameter name below.

Reply:
fixed